src/ontology/mondo-edit.obo

```diff
@@ -151299,7 +151225,7 @@ synonym: "Congenital dyserythropoietic anaemia type 3" EXACT [DOID:0111399]
 synonym: "congenital dyserythropoietic anaemia type III" EXACT [DOID:0111399]
 synonym: "congenital dyserythropoietic anemia type 3" EXACT [Orphanet:98870]
 synonym: "Congenital dyserythropoietic anemia type III" EXACT [Orphanet:98870]
-synonym: "congenital dyserythropoietic anemia type III" EXACT [DOID:0111399]
```

Comment:
Why is the DOID provenance from this line `synonym: "congenital dyserythropoietic anemia type III" EXACT [DOID:0111399]` not moved to the line above (`synonym: "Congenital dyserythropoietic anemia type III" EXACT [Orphanet:98870]`) where the synonym is the same?